lib/instrument/transformer/nskeyed.js

```diff
@@ -0,0 +1,378 @@
+  };
+};
+
+UNARCHIVE_CLASS_MAP.NSError = function () {
+  this.decodeArchive = function (archive) {
+    const domain = archive.decode('NSDomain');
+    const userinfo = archive.decode('NSUserInfo');
+    const code = archive.decode('NSCode');
+    return {$class: 'NSError', domain, userinfo, code};
+  };
+};
+
+UNARCHIVE_CLASS_MAP.NSExceptionArchive = function () {
+  this.decodeArchive = function (archive) {
+    const name = archive.decode('NS.name');
+    const reason = archive.decode('NS.reason');
+    const userinfo = archive.decode('userinfo');
+    return {'$class': 'NSException', reason, userinfo, name};
+  };
+};
+
+UNARCHIVE_CLASS_MAP.NSMutableString = function () {
+  this.decodeArchive = function (archive) {
+    return archive.decode('NS.string');
+  };
+};
+
+UNARCHIVE_CLASS_MAP.NSMutableData = function () {
+  this.decodeArchive = function (archive) {
+    return archive.decode('NS.data');
+  };
+};
+
+UNARCHIVE_CLASS_MAP.NSUUID = function () {
+  this.decodeArchive = function (archive) {
+    const uuidBytes = archive.decode('NS.uuidbytes');
+    return uuidStringify(uuidBytes);
+  };
+  this.encodeArchive = function (obj, archive) {
+    archive._archive_obj['NS.uuidbytes'] = uuidParse(obj).buffer;
+  };
+};
+
+UNARCHIVE_CLASS_MAP.XCTCapabilities = function () {
+  this.decodeArchive = function (archive) {
+    return archive.decode('capabilities-dictionary');
+  };
+};
+
+UNARCHIVE_CLASS_MAP.DTTapMessageArchive = function () {
+  this.decodeArchive = function (archive) {
+    return archive.decode('DTTapMessagePlist');
+  };
+};
+
+UNARCHIVE_CLASS_MAP.ErrorArchive = function () {
+  this.decodeArchive = function (archive) {
+    return {'$class': 'NSError', 'domain': archive.decode('NSDomain'), 'userinfo': archive.decode('NSUserInfo'), 'code': archive.decode('NSCode')};
+  };
+};
+
+UNARCHIVE_CLASS_MAP.ExceptionArchive = function () {
+  this.decodeArchive = function (archive) {
+    return {'$class': 'NSException', 'reason': archive.decode('NS.reason'), 'userinfo': archive.decode('userinfo'), 'name': archive.decode('NS.name')};
+  };
+};
+
+UNARCHIVE_CLASS_MAP.XCTCapabilities = function () {
+  this.decodeArchive = function (archive) {
+    return archive.decode('capabilities-dictionary');
+  };
+};
+
+UNARCHIVE_CLASS_MAP.DTKTraceTapMessage = function () {
+  this.decodeArchive = function (archive) {
+    if (archive._object.$0) {
+      return archive.decode('$0');
+    } else {
+      return archive.decode('DTTapMessagePlist');
+    }
+  };
+};
+
+UNARCHIVE_CLASS_MAP.DTTapHeartbeatMessage = function () {
+  this.decodeArchive = DTTapMessagePlist;
+};
+UNARCHIVE_CLASS_MAP.DTSysmonTapMessage = function () {
+  this.decodeArchive = DTTapMessagePlist;
+};
+UNARCHIVE_CLASS_MAP.NSURL = NSURL;
+
+
+/**
+ * Capable of unpacking an archived object tree in the NSKeyedArchive format.
+ * Apple's implementation can be found here:
+ * https://github.com/apple/swift-corelibs-foundation/blob/master/Foundation/NSKeyedUnarchiver.swift
```

Comment:
it seems like they changed the branch name: https://github.com/apple/swift-corelibs-foundation/blob/main/Sources/Foundation/NSKeyedUnarchiver.swift
(other comment also should update `master` to `main`)